marconi-core/src/Marconi/Core/Indexer/LastEventIndexer.hs

```diff
@@ -0,0 +1,266 @@
+{-# LANGUAGE FlexibleContexts #-}
```

Comment:
This is basically a FileIndexer, but for the last event only, right? We couldn't extend FileIndexer?

Reply:
We could, and this is how it was done initially, but having it in its own module streamlines a lot the logic and the way we process the events. If we want to combine both (and we probably want to chat about it first), I would prefer it to be a separate ticket.

Reply:
No problem. I'll create a ticket